packages/app/src/debug/DebugLoading.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div
+    data-cy="debug-loading"
+    class="p-30px"
+  >
+    <div>
+      <div class="flex items-center">
+        <div class="rounded-full bg-gray-50 h-16px w-344px" />
+        <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+          .
+        </div>
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-120px" />
+      </div>
+
+      <div class="flex mt-15px items-center">
+        <div class="rounded-full bg-gray-50 h-16px w-72px" />
+        <div class="rounded-full bg-gray-50 h-16px ml-8px w-160px" />
+        <div class="rounded-full bg-gray-50 h-16px ml-8px w-72px" />
+        <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+          .
+        </div>
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-80px" />
+        <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+          .
+        </div>
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-80px" />
+        <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+          .
+        </div>
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-80px" />
+        <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+          .
+        </div>
+        <div class="rounded-full bg-gray-50 h-8px ml-8px w-80px" />
+      </div>
+    </div>
+    <div class="mt-20px">
+      <div class="border-t border-l border-r rounded-t-md flex bg-gray-50 border-0 border-gray-100 w-full py-20px px-15px justify-between items-center">
+        <div>
+          <div class="flex items-center">
+            <i-cy-dot-solid_x4 class="icon-light-gray-100" /><div class="rounded-full bg-gray-100 h-16px ml-8px w-200px" />
+          </div>
+          <div class="flex mt-20px items-center">
+            <div class="rounded-full bg-gray-100 h-8px w-180px" />
+            <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+              .
+            </div>
+            <div class="rounded-full bg-gray-100 h-8px ml-8px w-99px" />
+            <div class="font-semibold mb-8px ml-8px text-14px text-gray-100">
+              .
+            </div>
+            <div class="rounded-full bg-gray-100 h-8px ml-8px w-99px" />
+          </div>
+        </div>
+        <div>
+          <div class="rounded-full bg-gray-100 h-20px w-120px" />
+        </div>
+      </div>
+    </div>
+    <div
+      v-for="i in [1, 2, 3]"
+      :key="i"
+      class="bg-white border flex border-b-0 border-gray-100 w-full py-5 px-17px items-center overflow-hidden"
+      :class="{ 'border-b rounded-b-md': i === 3 }"
+    >
```

Comment:
I feel like this duplicates some of the "skeleton" section of `DebugEmptyStates` - appears to from the Percy snapshots, minus the different length row segments (see my percentage comment above)

Reply:
I broke some of this out into shared components to cut down on duplicated code. Let me know what you think